code/src/App.js

```diff
@@ -1,9 +1,22 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { Game } from 'components/Game';
+import fetchGame from './reducers/gameFetch';
+import { loading } from './reducers/load';
 
 export const App = () => {
+  const reducer = combineReducers({
+    gameFetch: fetchGame.reducer,
+    loading: loading.reducer
+  });
+
+  const store = configureStore({ reducer,
+    middleware: (getDefaultMiddleware) => getDefaultMiddleware() });
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <Provider store={store}>
+      <Game />
+    </Provider>
+  );
+};
```

Comment:
We really like the overall theme of your game - the images and colours, lotties and arrows looks so good!